EFCore.InMemory.HierarchyId.Test/DatabaseContextTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Linq;
+using Microsoft.EntityFrameworkCore.SqlServer.Test.Models;
+using Xunit;
+
+namespace Microsoft.EntityFrameworkCore.SqlServer
+{
+    public class DatabaseContextTests
+    {
+        [Fact]
+        public void Should_CreateDatabaseContext()
+        {
+            // Arrange
+            var databaseContext = new AbrahamicContext();
+
+            // Act
+            databaseContext.Database.EnsureCreated();
+
+            // Assert
+            Assert.False(databaseContext == null);
```

Comment:
```suggestion
            Assert.NotNull(databaseContext);
```
and likewise in other places.

Reply:
Updated in https://github.com/efcore/EFCore.SqlServer.HierarchyId/pull/33/commits/1ebce664bff110512c259b99a1db2bae95d1c62f